mysql-test/include/percona_userstat_conn_handling.inc

```diff
@@ -71,6 +71,63 @@ connection default;
 --let $assert_cond= "$query_result" = 0
 --source include/assert.inc
 
+# Make a single connection for mysqltest_1 user
+connect (conn1,localhost,mysqltest_1,,);
+SELECT 1;
+connection default;
+
+# Verify that flush statistics keeps concurent connections counter inact
+FLUSH USER_STATISTICS;
+
+disconnect conn1;
+let $count_sessions= 1;
+--source include/wait_until_count_sessions.inc
+
+# verify flush user statistics with existing connections
```

Comment:
This comment is not addressed yet (but connection default should be added immediately after disconnect conn1 above)